ndcube/extra_coords/table_coord.py

```diff
@@ -10,12 +10,71 @@
 from astropy.coordinates import SkyCoord
 from astropy.modeling import models
 from astropy.modeling.models import tabular_model
+from astropy.modeling.tabular import _Tabular
 from astropy.time import Time
 from astropy.wcs.wcsapi.wrappers.sliced_wcs import combine_slices, sanitize_slices
 
 __all__ = ['TimeTableCoordinate', 'SkyCoordTableCoordinate', 'QuantityTableCoordinate']
 
 
+class Length1Tabular(_Tabular):
+    _input_units_allow_dimensionless = True
+    _has_inverse_bounding_box = True
+    _separable = True
+
+    n_inputs = 1
+    n_outputs = 1
+
+    lookup_table = np.zeros([1])
+    points = np.zeros([1])
+
+    def __init__(self, points=None, lookup_table=None, pixel_width=None,
```

Comment:
`pixel_width` is rather specific given the rest of this model allows anything in the points and lookup_table. `point_width` might be more generic?

Reply:
I'm open to the name change.  To clarify though, `pixel_width` is effectively CDELT, i.e it's in real world coords.  Is there a better name than `pixel_width` or `point_width` given this?

Reply:
ah, I see. The question here really becomes do we want these models to be only for a pixel <> world transform? Because at the moment, by virtue of inheriting `_Tabular` it isn't restricted to that.

I would say `value_width` would be better as you have values inside the lookup_table and points which are mapped to those values.